src/DoctrineBatchUtils/BatchProcessing/Exception/MissingBatchItemException.php

```diff
@@ -19,12 +19,17 @@ class MissingBatchItemException extends UnexpectedValueException implements Exce
 {
     public static function fromInvalidReference(ClassMetadata $metadata, object $object): MissingBatchItemException
     {
+        $identifier = json_encode($metadata->getIdentifierValues($object));
```

Comment:
Not a problem here, but `JSON_THROW_ON_ERROR` in future: let it crash and burn ;-)